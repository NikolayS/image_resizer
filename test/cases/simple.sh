#!/bin/bash

# to run this scenario, COPIERSERVICE should be defined (`export COPIERSERVICE=https://your.copier.host.name`)

http GET "$RESIZERSERVICE/?w=300&src=https://upload.wikimedia.org/wikipedia/commons/c/c2/Faust_bei_der_Arbeit.JPG" -h | grep -v "Date:"
