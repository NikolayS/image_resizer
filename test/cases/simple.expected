HTTP/1.1 200 OK
Server: nginx
Content-Type: image/jpeg
Transfer-Encoding: chunked
Connection: keep-alive
X-Content-Length-Original: 1367120
Content-Lenght: 14575

